jarviscli/plugins/news.py

```diff
@@ -123,8 +127,8 @@ def configure(self, jarvis):
         """
             configures the news channel of the user
         """
-        for index in self.source_dict.keys():
-            jarvis.say(str(index) + ": " + self.source_dict.get(index))
+        for index in sorted([ int(x) for x in self.source_dict.keys()]):
```

Comment:
```suggestion
        for index in sorted([int(x) for x in self.source_dict.keys()]):
```

Reply:
have fixed them already.. new build underway.. thanks for pointing out :)